### PR TITLE
Link selectors between charts and tables

### PR DESCRIPTION
**Pull request by tgsoverly**
Be able to reference one selector from one chart and have it change another charts or tables selection.

Note: If you reference another selector, it hides it on this chart.

https://github.com/visivo-io/visivo/assets/482572/9680e64f-5dee-42ae-8c96-db14692b5300

### CODE REVIEW

File: test-projects/demo/dashboards/welcome.visivo.yml [hunk 1]
```diff
@@ -60,6 +60,9 @@ dashboards:
           - width: 2
             chart:
               name: Fibonacci Waterfall Chart
+              selector:
+                name: Fibonacci Waterfall Selector
+                type: single
```
Comment (jpjsonic) [inline test-projects/demo/dashboards/welcome.visivo.yml:65]: Nice this really slick. Super easy to configure.

File: visivo/models/selector.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
+from enum import Enum
+from pydantic import ConfigDict, Field, PrivateAttr, model_serializer
+
+from visivo.models.base.base_model import BaseModel
+from visivo.models.base.named_model import NamedModel
+
+
+class SelectorType(str, Enum):
+    single = "single"
```
Comment (jpjsonic) [inline visivo/models/selector.py:9]: What's the default selection if you choose single? I don't know if it makes sense to do in this pass but we should allow people to specify a default selection(s) for both single and multi. I guess single would be one string and multi would be a list of strings maybe?

File: visivo/models/selector.py [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    multiple = "multiple"
+
+
+class Selector(NamedModel, BaseModel):
+    """
+    Selectors can be used to add interactivity between charts and tables.
+
+    Below is how you would link two charts to show the same selected data.
```
Comment (jpjsonic) [inline visivo/models/selector.py:17]: Yay for docs! We could add a a gif of your six second video in the docs too.

**Review by jpjsonic — APPROVED** [review]
This is awesome!

File: visivo/models/selector.py (hunk 2 above)
Comment (tgsoverly) [reply visivo/models/selector.py:9]: It grabs the "first" one.  Yea I agree with that!

File: visivo/models/selector.py (hunk 3 above)
Comment (tgsoverly) [reply visivo/models/selector.py:17]: Does the video live in the docs here or on "interactivity"?

**Pull request merged**